tests/test_menu.py

```diff
@@ -1,112 +1,160 @@
 
-    config = menu.configure_game()
+    config = menu.start()
 
     assert config.first_player.get_marker() == '!'
 
 
 def test_a_player_will_continue_to_be_prompted_if_they_provide_whitespace_as_a_marker():
-    menu = Menu(Console(MockConsoleIO(['1', ' ', ' ', '   ', '!', '', '1', '', '', '1'])))
+    menu = Menu(Console(MockConsoleIO(['1', '1', ' ', ' ', '   ', '!', '', '1', '', '', '1'])))
 
-    config = menu.configure_game()
+    config = menu.start()
 
     assert config.first_player.get_marker() == '!'
 
 
 def test_a_player_will_retain_their_default_marker_if_they_provide_an_empty_string():
-    menu = Menu(Console(MockConsoleIO(['1', '', '', '1', '', '', '1'])))
+    menu = Menu(Console(MockConsoleIO(['1', '1', '', '', '1', '', '', '1'])))
 
-    config = menu.configure_game()
+    config = menu.start()
 
     assert config.first_player.get_marker() == 'O'
 
 
 def test_if_player_one_selects_x_as_marker_player_two_default_marker_changes_to_o():
-    menu = Menu(Console(MockConsoleIO(['1', 'X', '', '1', '', '', '1', 'n'])))
+    menu = Menu(Console(MockConsoleIO(['1', '1', 'X', '', '1', '', '', '1', 'n'])))
 
-    config = menu.configure_game()
+    config = menu.start()
 
     assert config.second_player.get_marker() == 'O'
 
 
 def test_user_cannot_select_the_same_marker_as_their_opponent():
-    menu = Menu(Console(MockConsoleIO(['1', 'X', '', '2', 'X', 'O', '', '1'])))
+    menu = Menu(Console(MockConsoleIO(['1', '1', 'X', '', '2', 'X', 'O', '', '1'])))
 
-    config = menu.configure_game()
+    config = menu.start()
 
     assert config.second_player.get_marker() == 'O'
 
 
 def test_user_can_reverse_the_order_of_turns():
-    menu = Menu(Console(MockConsoleIO(['1', 'X', '', '2', 'O', '', '2'])))
+    menu = Menu(Console(MockConsoleIO(['1', '1', 'X', '', '2', 'O', '', '2'])))
 
-    config = menu.configure_game()
+    config = menu.start()
 
     assert config.first_player.get_name() == 'Player 2'
 
 
 def test_console_is_cleared_after_each_message_is_printed():
-    inputs = ['1', 'X', '', '2', 'O', '', '2', 'n']
+    inputs = ['1', '1', 'X', '', '2', 'O', '', '2', 'n']
     console = MockConsole(inputs)
     menu = Menu(console)
 
-    menu.configure_game()
+    menu.start()
 
     assert console.clear_output_call_count == len(inputs) - 1
 
 
+def test_user_can_choose_to_load_an_existing_game(json_data):
+    inputs = ['2', '1']
+    persister_io = MockPersisterIO(json_data)
+    persister = Persister(persister_io)
+    console = MockConsole(inputs)
+    menu = Menu(console, persister=persister)
+
+    config = menu.start()
+
+    assert config.first_player.get_name() == "Player 1"
+    assert config.second_player.get_name() == "Player 2"
+    assert config.board.get_spaces() == ["X", "O", "X", "O", "5", "6", "7", "8", "9"]
+
+
+def test_user_will_continue_to_be_prompted_if_they_provide_an_invalid_save_id(json_data):
+    inputs = ['2', '2', '5', '1']
```

Comment:
I see a mixture of single and double quotes—which are preferable in Python for this type of string?

Reply:
Or is there a preference?

Reply:
The python docs say 'String literals can be enclosed in matching single quotes (') or double quotes (").' which suggests to me that there's no difference, but reading around, the suggested approach seems to be that double quotes are to be used for natural language strings and single quotes for  anything else like dictionary strings, regex etc. 

Given that's the case I think single quotes are correct here. I guess the most important thing is consistency so I'll check through to see that I'm using the same style throughout - thanks for picking this up!